### ISSUE TITLE
Automatic support for alphanumeric versioning

### ISSUE THREAD

**Issue by TimidScript** [JasonBarnabe/greasyfork#371]
Consider allowing alphanumeric characters in version meta. It's not uncommon to have versions like 1.a or 1.2-rc1 and such.

**Comment by JasonBarnabe** [comment]
Alphanumerics are supported. There's a number of examples in https://github.com/JasonBarnabe/greasyfork/blob/master/test/models/script_version_compare_numbers_test.rb#L10

What specifically is not working for you?

**Comment by TimidScript** [comment]
>"You updated the code but didn't increase the @version number. Those who previously installed your script may not get the updated code.Save anyway"

The message above comes up a lot even if version numbers aren't the same. I just tested it and sometimes it comes up and other times it doesn't. On each case I changed the version letter at the end (always higher).

Example I have version 3.0.26 on a test script and I increased it to 3.0.26a it give me the message: But then I upgraded from 3.0.28 to 3.0.38c and then 3.0.38d, I didn't get the message. I don't grasp when the warning is fired. 



The initial number is same but the character can go to or rc-2 or beta2
